org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/commands/ProjectCommand.java

```diff
@@ -60,6 +61,9 @@
 public class ProjectCommand {
 
 	public static final String VM_LOCATION = IConstants.PLUGIN_ID + ".vm.location";
+	public static final String SOURCE_PATHS = IConstants.PLUGIN_ID + ".sourcePaths";
+	public static final String DEFAULT_OUTPUT_PATH = IConstants.PLUGIN_ID + ".defaultOutputPath";
```

Comment:
At first glance, i didn't get the point of `defaultOutputPath`. This is not aligned with other names such as "sourcePaths".

I would just rename the key to `.outputPath`, and leave its actual meaning in the comment.